app/src/main/java/nerd/tuxmobil/fahrplan/congress/MyApp.java

```diff
@@ -1,21 +1,16 @@
 package nerd.tuxmobil.fahrplan.congress;
```

Comment:
:guide_dog: Commit message: Please add a basic flow of the update mechanism to your commit message which would be easier to comprehend than studying the commit diff. Here is an example:

> Remove MyApp.lectureList
> 
> Replace the various mechanisms to get FahrplanFragment to update
> the lecture list with just one mechanism.
>
> UI -> AppRepository#notifyHighlightsChanged -> OnLecturesChangeListener -> FahrplanFragment
> UI -> AppRepository#notifyAlarmsChanged -> OnLecturesChangeListener -> FahrplanFragment
>
> That mechanism should be removed once AppRepository is an observable
> data source and changes of a Lecture will automatically cause an updated
> ScheduleData instance to be pushed to consumers.